packages/gestalt/src/PageHeader.js

```diff
@@ -0,0 +1,54 @@
+// @flow strict
+import React, { type Node, Children, type Element } from 'react';
+import Box from './Box.js';
+import Button from './Button.js';
+import IconButton from './IconButton.js';
+import Link from './Link.js';
+import Tooltip from './Tooltip.js';
+import Flex from './Flex.js';
+import Heading from './Heading.js';
+import Text from './Text.js';
+import { type Dimension } from './boxTypes.js';
+
+type Props = {|
+  title: string,
+  maxWidth?: Dimension,
+  primaryAction?: Element<typeof Button | typeof IconButton | typeof Link | typeof Tooltip>,
+  secondaryAction?: Element<typeof Button | typeof IconButton | typeof Link | typeof Tooltip>,
+  subtext?: string,
+|};
+
+export default function PageHeader({
+  maxWidth = '100%',
+  primaryAction,
+  secondaryAction,
+  subtext,
+  title,
+}: Props): Node {
+  return (
+    <Box color="white" paddingX={8} width="100%">
+      <Flex flex="grow" justifyContent="center" maxWidth="100%">
+        <Flex justifyContent="between" alignItems="center" width={maxWidth}>
+          <Box marginEnd={4} minWidth={0} marginTop={2} marginBottom={2}>
```

Comment:
`minWidth={0}` ? what is it doing?

Reply:
just curious

Reply:
It's needed to ensure that truncation happens with Flex https://css-tricks.com/using-flexbox-and-text-ellipsis-together/

Reply:
Nice trick. If it's a well-known thing I guess ain't needed but maybe adding  a comment to it might help to document the intention of `minWidth={0}` and the expected behavior.